src/tools/recipe2plan.ts

```diff
@@ -12,24 +12,36 @@ import {StorageKeyRecipeResolver} from './storage-key-recipe-resolver.js';
 import {PlanGenerator} from './plan-generator.js';
 import {Flags} from '../runtime/flags.js';
 import {assert} from '../platform/assert-node.js';
+import {encodePlansToProto} from './manifest2proto.js';
 
+export enum OutputFormat { Kotlin, Proto }
 
 /**
  * Generates Kotlin Plans from recipes in an arcs manifest.
  *
  * @param path path/to/manifest.arcs
  * @return Generated Kotlin code.
  */
-export async function recipe2plan(path: string): Promise<string> {
+export async function recipe2plan(
+    path: string,
+    format: OutputFormat,
+    recipeFilter: string | undefined = undefined): Promise<string | Uint8Array> {
```

Comment:
I think you can just do this: `recipeFilter?: string`

Reply:
Done